From b6370991cc254737d07d04d2dca4fef1215e2536 Mon Sep 17 00:00:00 2001
From: Johnny Hughes <johnny@centos.org> 
Date: Tue, 15 Dec 2015 12:26:48 -0600
Subject: [PATCH] centos grub2-password fix

---
 util/grub-setpassword.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util/grub-setpassword.in b/util/grub-setpassword.in
index dd76f00..a322dfb 100755
--- a/util/grub-setpassword.in
+++ b/util/grub-setpassword.in
@@ -1,7 +1,7 @@
 #!/bin/sh -e
 
 if [ -d /sys/firmware/efi/efivars/ ]; then
-    grubdir=`echo "/@bootdirname@/efi/EFI/redhat/" | sed 's,//*,/,g'`
+    grubdir=`echo "/@bootdirname@/efi/EFI/centos/" | sed 's,//*,/,g'`
 else
     grubdir=`echo "/@bootdirname@/@grubdirname@" | sed 's,//*,/,g'`
 fi
-- 
1.8.3.1

