From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Thu, 21 Jun 2018 19:09:11 -0400
Subject: [PATCH] tftp: Fix a minor const error

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 grub-core/net/tftp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/grub-core/net/tftp.c b/grub-core/net/tftp.c
index 17583cba380..e267af354f4 100644
--- a/grub-core/net/tftp.c
+++ b/grub-core/net/tftp.c
@@ -307,7 +307,7 @@ static void
 grub_normalize_filename (char *normalized, const char *filename)
 {
   char *dest = normalized;
-  char *src = filename;
+  const char *src = filename;
 
   while (*src != '\0')
     {
